### PR TITLE
aws/eks_service: add variables to manage create namespace

### PR DESCRIPTION
**Pull request by spartan-haopham**
## Summary
* Add `var.create_kubernetes_namespace` whether to create namespace
* Add `var.secret_suffix` to prevent hardcode the suffix

## Types of Changes

- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 🕷 Bug fix (non-breaking change which fixes an issue)
- [ ] 👏 Performance optimization (non-breaking change which addresses a performance issue)
- [ ] 🛠 Refactor (non-breaking change which does not change existing behavior or add new functionality)
- [ ] 📗 Library update (non-breaking change that will update one or more libraries to newer versions)
- [x] 📝 Documentation (non-breaking change that doesn't change code behavior, can skip testing)
- [ ] ✅ Test (non-breaking change related to testing)
- [ ] 🔒 Security awareness (changes that effect permission scope, security scenarios)

## Related Issues
https://www.notion.so/c0x12c/Infra-Rollout-the-new-Terraform-modules-Helm-to-Regimen-16401fb05bf1803ca209e1a7c0a96520?pvs=4

### CODE REVIEW

File: aws/eks-service/variables.tf [hunk 1]
```diff
@@ -37,3 +37,15 @@ variable "eks_oidc_provider" {
     url = string
   })
 }
+
+variable "suffix_service_name" {
```
Comment (spartan-toannhan) [inline aws/eks-service/variables.tf:41]: please rename to `suffix_env_var`

Comment (spartan-tainguyen) [inline aws/eks-service/variables.tf:41]: should it be something like secret_suffix instead?

File: aws/eks-service/variables.tf [hunk 2]
```diff
@@ -37,3 +37,15 @@ variable "eks_oidc_provider" {
...
+  description = "To custom name suffix for the kubernetes secret"
+  type        = string
+  default     = "env-var"
+}
+
+variable "create_kubernetes_namespace" {
+  description = "To specify whether to create a namespace"
+  type        = bool
+  default     = false
+}
```
Comment (spartan-tainguyen) [inline aws/eks-service/variables.tf:51]: EOF

File: aws/eks-service/variables.tf (hunk 1 above)
Comment (spartan-tainguyen) [reply aws/eks-service/variables.tf:41]: duplicated

**Review by spartan-toannhan — APPROVED** [review]

**Pull request merged**